arch/x86/core/x86_mmu.c

```diff
@@ -29,6 +29,12 @@ MMU_BOOT_REGION((u32_t)&__app_ram_start, (u32_t)&__app_ram_size,
 MMU_BOOT_REGION((u32_t)&_app_smem_start, (u32_t)&_app_smem_size,
 		MMU_ENTRY_WRITE | MMU_ENTRY_USER | MMU_ENTRY_EXECUTE_DISABLE);
 #endif
+
+#ifdef CONFIG_COVERAGE_HW_TEST
+MMU_BOOT_REGION((u32_t)&__gcov_bss_start, (u32_t)&__gcov_bss_size,
+		MMU_ENTRY_WRITE | MMU_ENTRY_USER | MMU_ENTRY_EXECUTE_DISABLE);
```

Comment:
I know that everything here is using `u32_t` to pointers, so it's just a question. @nashif is there a reason to not being using `uintptr_t` ?

Reply:
@nashif ^